scipy/stats/_continuous_distns.py

```diff
@@ -5160,6 +5160,9 @@ class logistic_gen(rv_continuous):
 
     `logistic` is a special case of `genlogistic` with ``c=1``.
 
+    Remark that the survival function (`sf`) is equal to the Fermi-Dirac
+    distribution describing occupation levels in spin-half physics.
```

Comment:
Ugh, surely not only spin-1/2. I'd suggest the code word non-intetacting fermions or free fermions. Or just say fernionic statistics and leave it at that.

Reply:
Yeah, you are right. I generally just consider spin-half as everything with an integer + half ;)

I'll amend.

Reply:
Just focus on fermions, not spins (half-integer or not), I'd say. We probably don't want to mix up things like Heisenberg like spin models etc :-).

Reply:
I think the latest one is fine, only mentioning `fermionic statistics`